metadata.json

```diff
@@ -9,8 +9,9 @@
   "issues_url": "https://tickets.puppetlabs.com/browse/MODULES",
   "dependencies": [
     {"name":"puppetlabs/stdlib","version_requirement":">= 4.6.0 < 5.0.0"}
```

Comment:
Ah, good question.

https://github.com/puppetlabs/puppetlabs-ntp/blob/6331180a9988d75e1aabdd924a88903a1adf9b01/manifests/params.pp#L2 seems to be the last stdlib usage, and that looks like something that needs reevaluation, when puppet3 support is dropped.